cmd/node/node_test.go

```diff
@@ -981,8 +981,7 @@ func getTestDefaultConfig() *config.Config {
 	// note: these need to be set sufficiently low enough that turbohare finishes well before the LayerDurationSec
 	cfg.HARE.RoundDuration = 2
 	cfg.HARE.WakeupDelta = 1
-	cfg.HARE.N = 5
-	cfg.HARE.F = 2
+	cfg.HARE.N = 6
```

Comment:
To add to the discussion: here majority was 3 (`N/2 + 1`) and `F` was 2.
With `N = 6` majority is now 4 - which seems correct to me.